### PR TITLE
[ensure_git_branch] Add a regex example

### PR DESCRIPTION
**Pull request by petkrein**
### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [ ] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by max-ott** [comment]
Looking good @petkrein Could you please check with regards to the CLA?

**Comment by petkrein** [comment]
@googlebot I signed it!

**Comment by petkrein** [comment]
@max-ott cla checked!

File: fastlane/lib/fastlane/actions/ensure_git_branch.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :branch,
                                        env_name: "FL_ENSURE_GIT_BRANCH_NAME",
-                                       description: "The branch that should be checked for. String that can be either the full name of the branch or a regex to match",
+                                       description: "The branch that should be checked for. String that can be either the full name of the branch or a regex e.g. '^feature\/.*$' to match",
```
Comment (janpio) [inline fastlane/lib/fastlane/actions/ensure_git_branch.rb]: Hm, would using \` instead of ' here maybe format the string as code in the docs output?

Comment (petkrein) [reply fastlane/lib/fastlane/actions/ensure_git_branch.rb]: i agree

**Review by janpio — APPROVED** [review]

**Review by joshdholtz — APPROVED** [review]
🔥 Thanks for the contribution! ❤️

**Pull request merged**